docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -92,3 +92,28 @@ statements, the connector supports the following features:
 - {doc}`/sql/alter-table`
 - {doc}`/sql/create-schema`
 - {doc}`/sql/drop-schema`
+
+## Performance
+
+The connector includes a number of performance improvements, detailed in the
+following sections.
+
+(snowflake-pushdown)=
+### Pushdown
+
+The connector supports pushdown for a number of operations:
+
+- {ref}`join-pushdown`
```

Comment:
I looked some more at the code .. pretty sure all these three are not supported either.. so I am removing these